src/controllers/selectedAccount/selectedAccount.ts

```diff
@@ -317,10 +321,40 @@ export class SelectedAccountController extends EventEmitter {
     }
   }
 
+  get oldAccountBanner(): Banner[] {
```

Comment:
Minor: I'd call this `deprecatedSmartAccountAccountBanner` or something, because "old" is too generic (v1 accs are also old).